example/ck_tile/03_gemm/instances/gemm_api.cpp

```diff
@@ -0,0 +1,480 @@
+                            ALayout_,
+                            BLayout_,
+                            CLayout_,
+                            M_Tile_,
+                            N_Tile_,
+                            K_Tile_,
+                            M_Warp_,
+                            N_Warp_,
+                            K_Warp_,
+                            M_Warp_Tile_,
+                            N_Warp_Tile_,
+                            K_Warp_Tile_,
+                            kPadM_,
+                            kPadN_,
+                            kPadK_>;
+
+float gemm(const gemm_traits& t, const ck_tile::GemmHostArgs& a, const ck_tile::stream_config& s)
+{
+    if(t.data_type.compare("fp16") == 0)
+    {
+        if(t.is_a_rowmajor && t.is_b_rowmajor && t.is_c_rowmajor)
+        {
+            if(a.M > 512)
+            {
+                // universal gemm compute bound RR
+                std::cout << "fp16 comp\n";
+                return gemm_<trait_<FP16,
+                                    FP16,
+                                    FP32,
+                                    FP16,
+                                    Row,
+                                    Row,
+                                    Row,
+                                    256,
+                                    256,
+                                    32,
+                                    2,
+                                    2,
+                                    1,
+                                    32,
+                                    32,
+                                    16,
+                                    false,
+                                    false,
+                                    false>>(a, s);
+            }
+            else
+            {
+                // universal gemm memory bound RR
+                std::cout << "fp16 mem\n";
+                return gemm_<trait_<FP16,
+                                    FP16,
+                                    FP32,
+                                    FP16,
+                                    Row,
+                                    Row,
+                                    Row,
+                                    128,
+                                    32,
+                                    64,
+                                    4,
+                                    1,
+                                    1,
+                                    32,
+                                    32,
+                                    8,
+                                    false,
+                                    false,
+                                    false>>(a, s);
+            }
+        }
+        else if(t.is_a_rowmajor && !t.is_b_rowmajor && t.is_c_rowmajor)
+        {
+            if(a.M > 512)
+            {
+                // universal gemm compute bound RC
+                std::cout << "fp16 comp RC\n";
```

Comment:
This should be logged only when stream_config is configured appropriately.

Reply:
I think we can just delete these since similar messages are being logged in different parts of code and this is a debugging artifact